Add draft minutes from 2022-02-10 (#93)
confidential-computing/governance: TAC/Meetings/2022/2022-02-10/CCC_TAC_Minutes-2022-02-10.md

# Confidential Computing Consortium TAC meeting

*The TAC meets for two hours on alternating Thursdays. All members of the community are welcome to attend and participate.*

## Meeting details

**Date**: Thursday, February 10, 2022

**Time**: 7a-9a US Pacific Time

* **Zoom**: [https://zoom.us/j/184384055](https://zoom.us/j/184384055) (pw: 43690)
* **Calendar**: [https://calendar.confidentialcomputing.io](https://calendar.confidentialcomputing.io) ([ical](https://calendar.google.com/calendar/ical/c_c0pcihr7n2n1k3a38i32d9ag10%40group.calendar.google.com/public/basic.ics), [Google Calendar](https://calendar.google.com/calendar/u/0/r?cid=c_c0pcihr7n2n1k3a38i32d9ag10@group.calendar.google.com))
* **Recording**: [https://www.youtube.com/watch?v=aa452MBVr44](https://www.youtube.com/watch?v=aa452MBVr44)

### Voting members of the TAC

*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*

* **Accenture** - Giuseppe Giordano

> * **ANT Group** - Zongmin Gu
> * **ARM** - Thomas Fossati / Michael Lu

Reviewer: ```suggestion
* **Ant Group** - Zongmin Gu
* **Arm** - Thomas Fossati / Michael Lu
```

* **LF Training course on DEI**: [Inclusive Open Source Community Orientation (LFC102) (free)](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)
* **Declared project dependencies**: [Google Sheets](https://docs.google.com/spreadsheets/d/1UKnbbGWXYLjnPZsox3zmYo59nv3XSXjePfas5E2fER0/edit#gid=0)
* **YouTube**: [youtube.confidentialcomputing.io](https://youtube.confidentialcomputing.io)
* **LFX**: [lfx.linuxfoundation.org](https://lfx.linuxfoundation.org)
* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)

## Agenda and Minutes

### Welcome

Rep. Thaler opened the call at 7:06a PT. He welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation.

### Roll Call / Attendance

#### Voting members

* Dan Middleton (Intel)
* Dave Thaler (Microsoft)
* Lily Sturman (Red Hat)
* Thomas Fossati (ARM)
* Eric Northup (Meta)

#### Attendees

* Aeva Black (Microsoft)
* Brian Warner (Linux Foundation)
* Chaitan Shet (Linux Foundation)

> * Christopher Haster (ARM)

Reviewer: My understanding was that it's now capitalized Arm, as it appears at https://www.arm.com/ and https://www.arm.com/company/policies/trademarks

Author: ```suggestion
* Christopher Haster (Arm)
```

* Pradeep (IBM)
* Ravi Sahita (Rivos)
* Shale Xiong (ARM)
* Shalom Shefa (Cisco)
* Shubhra Kar (Linux Foundation)
* Simon Johnson (Intel)
* Stephen Walli (Microsoft)
* Steve Van Lare (Anjuna)
* Xinxin Fan (IoTeX)
* Zhipeng Huang (Huawei)

### Meeting minute approvals

#### Approved by voting members

**RESOLVED:** That the [minutes of the Jan. 27, 2022](https://lists.confidentialcomputing.io/g/main/files/TAC/Meetings/2022/01-Jan/CCC_TAC_Minutes-2022-01-27-update.pdf) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.

#### Pending approval

* [December 16, 2021](../../2021/2021-12-16/TAC_Minutes-2021-12-16.pdf), further updates needed
* [January 13, 2022](../2022-01-13/TAC_Minutes-2022-01-13.pdf)

### Open Action Items

1. **[Mike, Eric, Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.

1. **[Thomas F., Mike B.]** Attestation SIG co-chairs to discuss / clarify goals.

>   * Attestation SIG will be ready for an annual review within a month.

Author: ```suggestion
  * Attestation SIG will be ready for an annual review within a month.
  * Moved to main agenda.
```

    * Next step is to submit the project contribution agreement and IP assignment to progress toward board approval.
    * Howard Huang volunteered as mentor for Veraison.

### TAC Tech Talk - LF Mentorship

Nick Vidal (NV), Shubhra Kar, and Chaitan Shet gave an update on the LFX mentorship platform. Discussion ensued over the source of funding. Stephen Walli (SW) requested a future discussion on how LFX is being used specifically for the Enarx LFX Mentorship.

### Code of Conduct escalation

Dan Middleton (DM) and Aeva Black (AB) provided an update on Code of Conduct escalation paths. AB noted that the natural path of escalation is within a project using the project's own CoC process. They noted that if an issue requires escalation it should go to the CCC. If there are conflicts of interest or non-resolution, it would be escalated above the CCC. DM noted that there needs to be a defined escalation path. AB noted that it is a best practice to have such reports go to small group of defined recipients or to a single named individual, as opposed to a broad team such as the TAC.

> DT noted it is important to also measure the use of escalation paths, because they may either be not needed or unusable and the end result can appear the same. The absence of a metric is not the same as the absence of the problem.

Reviewer: I don't recall saying this, please confirm attribution.  Maybe it was someone else?   If you've confirmed it was me, then leave as is.

DM asked if a specific escalation process is required. AB noted there may be a need to have a specific set of individuals who are more knowledgeable of CCC dynamics who are centered in the process.

BW reviewed the process of escalation at the Linux Foundation when a report is escalated beyond a project. He noted that the LF almost always lacks specific context, and will involve the project community for context anyhow. In the interest of ensuring a functional process, he recommended the project specify its own escalation path with the Linux Foundation as fallback.

**ACTION:** Stephen Walli to work with Dan Middleton, Aeva Black, and others to develop an escalation process, to be presented to TAC and governing board.

**ACTION:** Dan Middleton to open a pull request adding a defined escalation process to project requirements.

### Process improvement suggestions

> BW reviewed a number of process improvement suggestions. Dave noted that process improvements shouldn't require a TAC vote. Consensus was reached that if there are improvements that will enhance the efficiency of the TAC, that they should be made transparency. 

Reviewer: ```suggestion
BW reviewed a number of process improvement suggestions. Dave noted that improvements to internal processes within the purview of the TAC Chair and/or LF PM shouldn't require a TAC vote. Consensus was reached that if there are improvements that will enhance the efficiency of the TAC, that they should be made transparently. 
```

> Jethro Beekman (JB) noted these are public meetings and everyone is welcome to attend, but there is still some distance between open meetings and recordings being broadly available to the whole internet. He noted this could create hesitance that would prevent people from participating in meetings. SW provided historical context, noting meeting recordings are the backup to the minutes. He reviewed historical concerns about posting recordings. DT noted there are two questions. Are they stored on YouTube? And are they private, unlisted, or public?

Reviewer: ```suggestion
Jethro Beekman (JB) noted these are public meetings and everyone is welcome to attend, but there is still some distance between open meetings and recordings being broadly available to the whole Internet. He noted this could create hesitance that would prevent people from participating in meetings. SW provided historical context, noting meeting recordings are the backup to the minutes. He reviewed historical concerns about posting recordings. DT noted there are two questions. Are they stored on YouTube? And are they private, unlisted, or public?
```
